react-components/src/components/CadModelContainer/useApplyCadModelStyling.tsx

```diff
@@ -0,0 +1,125 @@
+import { CogniteCadModel, DefaultNodeAppearance, NodeAppearance, NodeCollection, NodeIdNodeCollection, TreeIndexNodeCollection, IndexSet } from "@cognite/reveal";
+import { useEffect } from "react";
+import { CadModelStyling, useReveal } from "../..";
+import { NodeStylingGroup, TreeIndexStylingGroup, modelExists } from "./CadModelContainer";
```

Comment:
There is a circular dependency here. CadModelContainer depends on useApplyCadModelStyling, and vice versa.